hackathon_site/hardware/views.py

```diff
@@ -101,7 +109,10 @@ def get_permissions(self):
         if self.request.method == "POST":
             return [UserHasProfile()]
         if self.request.method == "GET":
-            return [FullDjangoModelPermissions()]
+            if self.request.user.has_perm("hardware.change_order"):
+                return [FullDjangoModelPermissions()]
```

Comment:
This requires the user to have view permissions and it throws a forbidden error if a user only has change permissions. I don't think this is a big deal, we just have to remember to give tech team view and change permissions.

Also I was trying to combine permissions with `and` and `or` but it wasn't working ://// I'll try to look into it more because this if statement is a bit messy